Fix mini typos in docs (#1600)
JuliaStats/Distributions.jl: docs/src/multivariate.md

logpdf(::MultivariateDistribution, ::AbstractArray)
loglikelihood(::MultivariateDistribution, ::AbstractVector{<:Real})
```

> **Note:** For multivariate distributions, the pdf value is usually very small or large, and therefore direct evaluation of the pdf may cause numerical problems. It is generally advisable to perform probability computation on a log scale.

Reviewer: Hmm, not sure about "on a log scale". I guess I would prefer the original version:
```suggestion
**Note:** For multivariate distributions, the pdf value is usually very small or large, and therefore direct evaluation of the pdf may cause numerical problems. It is generally advisable to perform probability computation in log scale.
```

Author: "on" is a much more popular preposition in this case, but happy to take your suggestion
